src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -0,0 +1,1261 @@
+
+  hysterectomy_margins_involved:
+    $ref:
+      - "_terms.yaml#/hysterectomy_margins_involved/common"
+    enum:
+      - Bladder
+      - Macroscopic Parametrium
+      - Microscopic Parametrium
+      - None
+      - Vagina
+      - Unknown
+      - Not Reported
+    enumDef:
+      None:
+        $ref:
+          - "_terms_enum.yaml#/none/common"
+      Unknown:
+        $ref:
+          - "_terms_enum.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  hysterectomy_type:
+    $ref:
+      - "_terms.yaml#/hysterectomy_type/common"
+    enum:
+      - Hysterectomy, NOS
+      - Not Performed
+      - Radical Hysterectomy
+      - Simple Hysterectomy
+      - Unknown
+      - Not Reported
+    enumDef:
+      Unknown:
+        $ref:
+          - "_terms_enum.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  immunosuppressive_treatment_type:
+    $ref:
+      - "_terms.yaml#/immunosuppressive_treatment_type/common"
+    enum:
+      - Anti-TNF Therapy
+      - Azathioprine
+      - Cyclophosphamide
+      - Methotrexate
+      - None
+      - Other
+      - Unknown
+      - Not Reported
+    enumDef:
+      None:
+        $ref:
+          - "_terms_enum.yaml#/none/common"
+      Other:
+        $ref:
+          - "_terms_enum.yaml#/other/common"
+      Unknown:
+        $ref:
+          - "_terms_enum.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  menopause_status:
+    $ref:
+      - "_terms.yaml#/menopause_status/common"
+    enum:
+      - Perimenopausal
+      - Postmenopausal
+      - Premenopausal
+      - Unknown
+      - Not Reported
+    enumDef:
+      Unknown:
+        $ref:
+          - "_terms_enum.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms_enum.yaml#/not_reported/common"
+
+  myasthenia_gravis_classification:
+    $ref:
+      - "_terms.yaml#/myasthenia_gravis_classification/common"
+    enum:
+      - Class I
+      - Class II
+      - Class III
+      - Class IV
+      - Class V
+    enumDef:
+      Class I:
+        $ref:
+          - "_terms_enum.yaml#/i/other_clinical_attribute/myasthenia_gravis_classification"
+      Class II:
+        $ref:
+          - "_terms_enum.yaml#/ii/other_clinical_attribute/myasthenia_gravis_classification"
+      Class III:
+        $ref:
+          - "_terms_enum.yaml#/iii/other_clinical_attribute/myasthenia_gravis_classification"
+      Class IV:
+        $ref:
+          - "_terms_enum.yaml#/iv/other_clinical_attribute/myasthenia_gravis_classification"
+      Class V:
+        $ref:
+          - "_terms_enum.yaml#/v/other_clinical_attribute/myasthenia_gravis_classification"
+
+  nadir_cd4_count:
+    $ref:
+      - "_terms.yaml#/nadir_cd4_count/common"
+    type: number
+    minimum: 0
+
+  nononcologic_therapeutic_agents:
+    description: >-
```

Comment:
for the new properties, any particular reason why we are not putting the descriptions in the _terms file instead?